proto/storage/virtual_machine.proto

```diff
@@ -47,3 +60,105 @@ message VirtualMachineScan {
   }
   repeated Note notes = 6;
 }
+
+// Next Tag: 14
+message EmbeddedVirtualMachineScanComponent {
+  string name = 1;
+  string version = 2;
+  message License {
+    string name = 1;
+    string type = 2;
+    string url = 3;
+  }
+  License license = 3;
+  repeated EmbeddedVirtualMachineVulnerability vulns = 4; // @gotags: hash:"set"
+  oneof has_layer_index {
+    int32 layer_index = 5;
+  }
+  int64 priority = 6; // @gotags: hash:"ignore"
+  enum SourceType {
+    OS = 0;
+    PYTHON = 1;
+    JAVA = 2;
+    RUBY = 3;
+    NODEJS = 4;
+    GO = 7;
+    DOTNETCORERUNTIME = 5;
+    INFRASTRUCTURE = 6;
+  }
+  SourceType source = 7;
+  string location = 8;
+  oneof set_top_cvss {
+    float top_cvss = 9;
+  }
+  float risk_score = 10; // @gotags: hash:"ignore"
+  // Component version that fixes all the fixable vulnerabilities in this component.
+  string fixed_by = 11;
+  message Executable {
+    string path = 1;
+    repeated string dependencies = 2;
+  }
+  // Values are cleared after moving to cache, remove them from the grpc return as well
+  repeated Executable executables = 12; // @gotags: json:"-"
+  string architecture = 13;
+}
+
+// Next Tag: 25
+message EmbeddedVirtualMachineVulnerability {
+  // ScoreVersion can be deprecated ROX-26066
+  enum ScoreVersion {
+    V2 = 0; // No unset for automatic backwards compatibility
+    V3 = 1;
+  }
+
+  enum VulnerabilityType {
+    UNKNOWN_VULNERABILITY = 0;
+    IMAGE_VULNERABILITY = 1;
+    K8S_VULNERABILITY = 2;
+    ISTIO_VULNERABILITY = 3;
+    NODE_VULNERABILITY = 4;
+    OPENSHIFT_VULNERABILITY = 5;
+  }
+
+  string cve = 1;
+  message Advisory {
+    string name = 1;
+    string link = 2;
+  }
+  Advisory advisory = 24;
+  float cvss = 2;
+  string summary = 3;
+  string link = 4;
+  oneof set_fixed_by {
+    string fixed_by = 5;
+  }
+  ScoreVersion score_version = 8;
+  // CVSSV2 CVSSV3 can be deprecated ROX-26066
+  CVSSV2 cvss_v2 = 6;
+  CVSSV3 cvss_v3 = 7;
+  google.protobuf.Timestamp published_on = 9;
+  google.protobuf.Timestamp last_modified = 10;
+  // For internal purposes only.
+  VulnerabilityType vulnerability_type = 11;
+  repeated VulnerabilityType vulnerability_types = 18; // @gotags: hash:"ignore"
+  bool suppressed = 12; // @gotags: hash:"ignore"
+  google.protobuf.Timestamp suppress_activation = 13; // @gotags: hash:"ignore"
+  google.protobuf.Timestamp suppress_expiry = 14; // @gotags: hash:"ignore"
```

Comment:
If we are not going to support suppression, might as well get these out of there.